src/styles/app/37-database.css

```
@@ -7,6 +7,115 @@
+	--database-control-radius: calc(var(--radius-md) + 1px);
+	--database-control-border: color-mix(
+		in srgb,
+		var(--border-default) 78%,
+		transparent
+	);
+	--database-control-background: color-mix(
+		in srgb,
+		var(--bg-primary) 92%,
+		var(--bg-secondary)
+	);
+	border-radius: var(--database-dropdown-radius);
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)